src/styles/main.scss

```diff
@@ -1,3 +1,55 @@
-body {
-  margin: 0;
+@import "page";
+@import "reset";
+@import "utils/variables";
+
+.page__content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  margin: 0 auto;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $size-days;
+
+  &__day {
+    position: relative;
+    padding: 10px;
+    height: $box-size;
+    width: $box-size;
+    background-color: #eee;
+    border: 1px solid #000;
+    cursor: pointer;
+  }
+
+  @for $length-day from 28 to 31 {
+    &--month--length--#{$length-day} :nth-child(n + #{$length-day + 1}) {
+      display: none;
+    }
+  }
+
+  @for $index from 1 to 32 {
```

Comment:
There isn't 32'th day in month)

Reply:
I'm not inclusive 32 day

Reply:
SASS loops include last number. It should be 31.